src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
+    }
+
+    @Override
+    public void stopWork() {
+        return "certain machines stopped working";
```

Comment:
Similarly, the 'stopWork' method should not return a string. It should print an informative message to the console indicating that the work has stopped, and specify the machine type, e.g., 'Excavator stopped working'.